packages/css-dev-tools/postcssPluginConfig.js

```diff
@@ -68,7 +68,7 @@ const plugins = [
 
 const productionPlugins = [
   cssprepend(`$mozaic-env: ${mozaicEnvScssVar};`),
-  nodeSass({
+  sass({
```

Comment:
You should remove the option `precision: 7,`, it's not useful when using Dart Sass.

=> More details here: https://github.com/sass/dart-sass#javascript-api

Reply:
cb07e64